backend-issuer/src/controllers/webhooks/index.ts

```diff
@@ -52,17 +56,25 @@ export class WebhooksController extends Controller {
     })
 
     if (payload.state === 'completed' && payload.invitationDid) {
+      let member
       try {
-        await this.db.updateMember(
+        member = await this.db.updateMember(
           {
             did: payload.invitationDid,
-            status: 'approved',
           },
-          { status: 'verified' }
+          { connectionId: payload.id }
         )
       } catch (e) {
-        throw new InternalError('Could not update status')
+        throw new InternalError('Could not update connectionId')
       }
+      log.info({
+        msg: `Connection ${payload.id} established with ${payload.invitationDid}`,
+      })
+      const { credDefId } = await this.db.getConfig()
```

Comment:
@msterle I might need you to quickly explain to me what is happening here

Reply:
The [Provisioner.provision()](https://github.com/digicatapult/nice-agent-portal/blob/main/backend-issuer/src/lib/provisioner.ts#L17-L40) method will always be run at server startup, ensuring we have `did`, `schemaId` and `credDefId` stored in both our Config database table and the associated resources defined in our Veritable storage.